modules/service-ca-certificates.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="service-ca-certificates_{context}"]
+= Service CA certificates
+
+== Purpose
+
+`service-ca` is an Operator that creates a self-signed CA, which is valid for one year,
+when an {product-title} cluster is deployed.
+
+== Expiration
+
+A custom expiration term is not supported. The self-signed CA is stored in a
+secret with qualified name `service-ca/signing-key` in fields `tls.crt`
+(certificate(s)), `tls.key` (private key), and `ca-bundle.crt` (CA bundle).
+
+Other services can request a service serving certificate by annotating a
+service resource with `service.beta.openshift.io/serving-cert-secret-name
+<secret name>`. In response, the Operator generates a new certificate, as
+`tls.crt`, and private key, as `tls.key` to the named secret. The certificate
+is valid for two years.
+
+Other services can request that the CA bundle for the service CA be injected
+into APIService or ConfigMap resources by annotating with
```

Comment:
Injection has been extended in 4.4 to include CustomResourceDefinition, MutatingWebhookConfiguration and ValidatingWebhookConfiguration.

An enhancement was recently merged that provides detail as to the implementation for these newly supported types: https://github.com/openshift/enhancements/blob/master/enhancements/authentication/service-ca-bundle-injection-for-admission-webhooks.md

Reply:
Thanks! Once this content merges, I will create a PR for these 4.4 changes and then backport the changes.